backend/config/rabbitmq.js

```diff
@@ -0,0 +1,27 @@
+const amqp = require("amqplib");        // Import RabbitMQ
+
+// Function to publish an order message to RabbitMQ
+async function publishOrder(order) {
+    try {
+        // Connect to RabbitMQ
+        const connection = await amqp.connect("amqp://localhost");
+        const channel = await connection.createChannel();
+
+        // Ensure the order queue exists
+        await channel.assertQueue("order_queue", { durable: true });
+
+        // Send the order message to the queue
+        channel.sendToQueue("order_queue", Buffer.from(JSON.stringify(order)), { persistent: true });
```

Comment:
why we just have 1 order queue? I thought we're supposed to have 2 queues 1 for buy and 1 for sell order